README.md

```diff
@@ -64,8 +64,7 @@ Para casos donde todo el video se clasifica de una clase y no pedazos de él, ha
 Para casos donde el video tiene partes de distintas clases.
 Acá importa para la clasificación cómo van apareciendo las clases y sus tiempos.
 
-0. [Reentrená la CNN](#cnn-inception).
-1. Tenés que ubicar los videos en la carpeta `rcnn/videos` y crear un archivo llamado `rcnn/classes.py` que defina el diccionario `class_per_frame` con el siguiente formato:
+0. Tenés que ubicar los videos en la carpeta `rcnn/videos` y crear un archivo llamado `rcnn/classes.py` que defina el diccionario `class_per_frame` con el siguiente formato:
```

Comment:
You can start from 1, the 0 is because it's something that was supposedly already done.

Reply:
I prefer to start with 0.